### PR TITLE
[COMDLG32] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to PR https://github.com/reactos/reactos/pull/5908

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: dll/win32/comdlg32/lang/cdlg_Ro.rc [hunk 1]
```diff
@@ -21,6 +21,15 @@
...
 
+/*
+ * PROJECT:     ReactOS DX Diagnostic
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATORS: Copyright 2008 Michael Stefaniuc
+ *              Copyright 2012 Ștefan Fulea <stefan.fulea@mail.com>
+ *              Copyright 2023 Andrei Miloiu <miloiuandrei@gmail.com>
+ */
+
```
Comment (binarymaster) [inline dll/win32/comdlg32/lang/cdlg_Ro.rc:32]:
```suggestion
```
(this is Wine file, modify existing header without reformatting it)

**Review by JoachimHenze — APPROVED** [review]
Approved. Didn't spot any accelerator collision in SAVE_FILE DIALOG.
But **I didn't check** for any accelerator collisions for IDS_SAVE_BUTTON within the STRINGTABLE.

**Comment by AndreiMiloiu** [comment]
I made some corrections. Please recheck.

**Pull request merged**